src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/ErrorReport.scala

```diff
@@ -0,0 +1,102 @@
+package org.broadinstitute.dsde.workbench.leonardo.model
+
+import akka.http.scaladsl.model.StatusCode
+import spray.json._
+
+import scala.language.postfixOps
+
+/*
+ * THIS CLASS NEEDS TO BE REMOVED ASAP
```

Comment:
Looks like it's already in `rawls/model`.. could we add rawls/model as a dependency to Leo?